combat-trainer.lic

```diff
@@ -1721,6 +1721,9 @@ class AttackProcess
     @offhand_thrown = settings.offhand_thrown
     echo("  @offhand_thrown: #{@offhand_thrown}") if $debug_mode_ct
 
+    @ambush_location = settings.ambush_location
```

Comment:
This needs an entry in `base.yaml`

I recommend defaulting it to `back`

Reply:
accounted for by https://github.com/rpherbig/dr-scripts/pull/1974/commits/c12852567a3e754a06716a664eac546b3f02350a